packages/core/src/components/way-input/way-input.e2e.ts

```diff
@@ -0,0 +1,42 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('way-input', () => {
+  it('should render', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<way-input></way-input>');
+
+    const element = await page.find('way-input');
+    expect(element).toHaveClass('hydrated');
+  });
+
+  it('should emit wayFocus when gaining focus', async () => {
+    const page = await newE2EPage({
+      html: `
+        <way-input></way-input>
+      `
+    });
+    const input = await page.find('way-input');
+    const wayFocus = await input.spyOnEvent('wayFocus');
```

Comment:
I think these spyOnEvent's now fail because of the new event naming (kebab-case instead of camelCase)

Reply:
Indeed, it's fixed now :)